src/entity.ts

```diff
@@ -991,16 +991,6 @@ export namespace entity {
     const MAX_DATASTORE_VALUE_LENGTH = 1500;
     if (Array.isArray(entities)) {
       for (const entry of entities) {
-        if (entry && entry.name && entry.value) {
-          if (
-            is.string(entry.value) &&
-            Buffer.from(entry.value).length > MAX_DATASTORE_VALUE_LENGTH
-          ) {
-            entry.excludeFromIndexes = true;
```

Comment:
Leaving this assignment in will apply `excludeFromIndexes: true` in more places than is necessary. As mentioned in the comment above, we already apply `excludeFromIndexes` in the right places elsewhere in the top level for arrays and in lower levels this code will actually apply `excludeFromIndexes: true` to the wrong places so it should be removed.